### PR TITLE
Fix resin preload when a commit is provided.

### PR DESCRIPTION
**Pull request by zvin**
Connects-To #629

Change-Type: patch

### CODE REVIEW

File: lib/actions/preload.coffee [hunk 1]
```diff
@@ -196,7 +196,7 @@ module.exports =
 					# Use the commit given as --commit or show an interactive commit selection menu
 					Promise.try ->
 						if options.commit
-							if _.map(builds, 'commit_hash').indexOf(options.commit) == -1
+							if _.map(application.build, 'commit_hash').indexOf(options.commit) == -1
```
Comment (dfunckt) [inline lib/actions/preload.coffee]: I think this can be

```
if _.find(application.build, commit_hash: options.commit)
  ...
```

Comment (emirotin) [inline lib/actions/preload.coffee]: Right, but with negation: `if !`

Comment (dfunckt) [inline lib/actions/preload.coffee]: Good catch, yes.

Comment (zvin) [inline lib/actions/preload.coffee]: :+1:

**Pull request merged**